api/app/graphql/mutations/auth/sign_in.rb

```diff
@@ -0,0 +1,31 @@
+module Mutations
+    module Auth
+      class SignIn < BaseMutation
+        require 'jwt_service'
```

Comment:
Since the `lib` folder is in the load path, it is not necessary to have this line here.